tests/CaptainHook/Storage/FileTest.php

```diff
@@ -92,11 +89,11 @@ public function testNoWritePermission()
     /**
      * Tests File::write
      */
-    public function testCantCreateDirectory()
+    public function testCantCreateDirectory(): void
     {
-        $this->expectException(\Exception::class);
+        $this->expectException(Exception::class);
 
-        $baseDir = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('basedir');
+        $baseDir = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('basedir', true);
```

Comment:
The second parameter improves the "randomness" of `uniqid` and is recommended to be used.